src/bitmessageqt/uisignaler.py

```diff
@@ -1,29 +1,39 @@
+"""
+src/bitmessageqt/uisignaler.py
+==============================
+"""
 
-from PyQt4.QtCore import QThread, SIGNAL
 import sys
 
+from PyQt4.QtCore import SIGNAL, QThread
+
 import queues
 
 
 class UISignaler(QThread):
+    """A QT thread for managing UI status updates"""
```

Comment:
This isn't really accurate. This thread processes incoming "signals" (technically queue entries) that other threads send to the UI.